### DISCUSSION TITLE
I can't figure out how to trigger a method in child component with a button in the parent component

### DISCUSSION THREAD

**Discussion by ozdemirrulass** [nuxt/nuxt#20418] [Questions] [answered]
Hi,

I am using Nuxt3 and as I indicated in the title I'm trying to trigger a method in the child component from the parent component can someone explain with a basic button and console log method ? 
Button should be in the parent component and the method with console log should be in the child component

thanks in advance

**Comment by ozdemirrulass** [comment, marked as answer]
Thanks to @dxphilo I solved the issue here is the article link if someone needs 

https://levelup.gitconnected.com/how-to-access-child-component-properties-in-vue-js-3-d47f6ae1d62

**Reply by dxphilo** [reply]
Thanks for finding the article helpful and sharing it

**Comment by eltonmorais** [comment]
A paid only answer should not be even allowed.

**Reply by dxphilo** [reply]
Here is a free-link. https://levelup.gitconnected.com/how-to-access-child-component-properties-in-vue-js-3-d47f6ae1d62?sk=b511fcc992b024e4e0b4e09a8f7bedcb
